### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mcarton**

### CODE REVIEW

File: clippy_lints/src/bit_mask.rs [hunk 1]
```diff
@@ -91,16 +91,11 @@ impl LateLintPass for BitMask {
...
-                                                                               right,
-                                                                               invert_cmp(cmp.node),
-                                                                               cmp_val,
-                                                                               &e.span)
-                                                             })
-                                                         },
-                                                         |cmp_opt| check_compare(cx, left, cmp.node, cmp_opt, &e.span))
+                if let Some(cmp_opt) = fetch_int_literal(cx, right) {
+                    check_compare(cx, left, cmp.node, cmp_opt, &e.span)
+                } else if let Some(cmp_val) = fetch_int_literal(cx, left) {
```
Comment (Manishearth) [inline clippy_lints/src/bit_mask.rs]: TIL else if let works

Comment (mcarton) [inline clippy_lints/src/bit_mask.rs]: :tada

**Comment by mcarton** [comment]
Rebased

**Comment by llogiq** [comment]
Good to have those readability improvements. Thanks @mcarton!

**Pull request merged**